README.md

```diff
@@ -2,40 +2,45 @@
 
 This module provides Sass and JavaScript utilities for reliably building paths to a module's static assets, such as CSS background images, fonts and JSON data files. This is needed because the URL path to load these assets will vary markedly depending on how a product developer chooses to integrate the Origami component into their website.
 
-## Browser support
-This module has been verified in Internet Explorer 7+, modern desktop browsers (Chrome, Safari, Firefox, …) and mobile browsers (Android browser, iOS safari, Chrome mobile).
+- [Overview](#overview)
+- [Usage](#usage)
+	- [JavaScript](#javascript)
+	- [Sass](#sass)
+	- [URL routing](#url-routing)
+	- [Using with the build service](#using-with-the-build-service)
+- [Contact](#contact)
+- [Licence](#licence)
 
-## Overview
 
-For example, if a module called `o-header` contains a stylesheet that loads a logo as a background, and that image exists at `/img/logo.png` in the module's git repository, the path that needs to be output in the CSS could vary wildly:
+# Overview
```

Comment:
Shouldn't these be `h2`s? This way they have the same heading level as the component name